### DISCUSSION TITLE
Installing OrcaSlicer Beta Updates with Homebrew

### DISCUSSION THREAD

**Discussion by leandrolima-nyc** [OrcaSlicer/OrcaSlicer#1689] [Dev]
I have a question regarding OrcaSlicer and its beta updates. Currently, the latest version available through Homebrew is 1.6.3. 

I was wondering if anyone here knows of a way to install beta updates of OrcaSlicer using Homebrew. I'd appreciate any insights or guidance you can provide on this matter.

Thank you!

**Comment by RaySajuuk** [comment]
It make sense to me, since 1.6.3 is the latest official release.

If you want to be on the beta or nightly train, I would suggest you clone this project and build your own, SoftFever has made it as simple as possible with included build_release_macos.sh. Which if you follow the "How to compile" in readme section you shouldn't have any issue building your own. You can go to the action section download the latest nightly binary.
